webapp/cypress/integration/common/login.spec.js

```diff
@@ -0,0 +1,150 @@
+import { Then, When, Given, And } from 'cypress-cucumber-preprocessor/steps'
+
+beforeEach(() => {
+  Given('I am on the site homepage', () => {
+    cy.visit('http://localhost:3000/')
+  })
+  When('I click on "Login" button on the "Home" Page', () => {
+    cy.get('button').contains('Login').click()
+  })
+  Then('I am redirected to "Login" Page', () => {
+    cy.location().should((loc) => {
+      expect(loc.pathname).to.eq('/login')
+    })
+  })
+})
+
+Given('I enter "admin@admin.pl" into "Email" field on the "Login" Page', () => {
+  cy.get('input[name=email]').type('admin@admin.pl')
+})
+And('I enter "admin" into "Password" field on the "Login" Page', () => {
+  cy.get('input[name=password').type('admin')
+})
+When('I click on "Login" button on the "Login" Page', () => {
+  cy.get('button').contains('Login').click()
+})
+Then('I am redirected to "History" Page of "admin@admin.pl" User', () => {
+  cy.location().should((loc) => {
+    expect(loc.pathname).to.eq('/panel/history')
+  })
+})
+And(
+  'Alert "Login successful" is displayed on "History" Page of "admin@admin.pl" User',
+  () => {
+    cy.on('window:alert', (str) => {
+      expect(str).to.equal('Login successful')
+    })
+  }
+)
+
+Given('I left empty "Email" field on the "Login" Page', () => {})
+And('I left empty "Password" field on the "Login" Page', () => {})
```

Comment:
Nie wiem czy te kroki są nam potrzebne w sumie, bo nic nie wnoszą do testu, a można się domyślić że jak nic nie wpisano to znaczy że nic tam nie ma.